### DISCUSSION TITLE
Global Gampus discount suddenly expired, unable to redeem benefits as usual

### DISCUSSION THREAD

**Discussion by qijiabao4113** [community/community#68971] [GitHub Education] [answered]
### Hi everyone,

I applied for a student permit in March. I received the benefits on the same afternoon. However, on August 30th of this year (2023), the benefits suddenly became invalid. I am unable to access any student benefits and cannot submit any new applications.
![image](https://github.com/community/community/assets/76029322/e408fa1a-0f9b-426b-a247-5b9c92f96451)
I attempted to resubmit my information and reapply for the Global Gamepus privileges, but the system does not allow me to do so because I already have student eligibility. However, the issue I am currently facing is that despite having student eligibility, the Global Gamepus privileges suddenly became invalid and have not been restored.
![image](https://github.com/community/community/assets/76029322/608e4b93-0b68-43da-b6e1-133231a33533)
It is now October 2, 2023, and my Global Gamepus privileges have been inactive for over a month. I feel helpless as I have been unable to reapply and unable to find any relevant solutions. I have not received any notifications or emails regarding this issue. What should I do to restore my Global Gamepus privileges? And who can assist me with this matter?who can help me, please?
![image](https://github.com/community/community/assets/76029322/db19ab08-ad84-4cf1-901e-bff93dcd986d)
This situation has been ongoing for a month with no changes or results.
![image](https://github.com/community/community/assets/76029322/4a4c8a37-8734-4c8b-9cab-1af410932e19)

**Comment by samus-aran** [comment, marked as answer]
Hi @qascetic 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by qijiabao4113** [reply]
:coffee:Thank you for your response. I truly appreciate your help in handling this matter. I will continue to monitor this discussion and if any further cooperation from me is needed, please leave a message for me. I will do my best to collaborate with you all to resolve this issue.:mag:

**Reply by daotrananhtuan09102002** [reply]
Hi, I have the same problem. Hope you fix it soon. Thank you.

**Reply by samus-aran** [reply]
Hi @qascetic 👋  Can we please ask you to try and re-apply. You should be able to proceed on re-verifying your academic status now 👍 

Hello too @daotrananhtuan09102002. We'll look into your situation too and if we have any updates we'll response on this thread.

**Reply by alu0101330105** [reply]
Im getting the same problem, I re-verified last 30 Sep and every thing seems to be in order and working, but today it all stopped working.

**Reply by qijiabao4113** [reply]
Thank you for your assistance. I am now able to resubmit my application. I have already attempted to resubmit it. I hope to receive a response to my application soon and I hope everything goes smoothly. Once again, I sincerely appreciate your help!

**Reply by samus-aran** [reply]
Hi @alu0101330105 & @daotrananhtuan09102002 👋 

Can we please ask you to try and re-apply. You should be able to proceed on re-verifying your academic status now 👍

**Reply by alu0101330105** [reply]
Thanks, i just did the re-verification. I'll come back and comment if the problem returns or if it fixed

**Reply by daotrananhtuan09102002** [reply]
Thank you for your help. I have just resubmitted it and am waiting for approval.

**Reply by qijiabao4113** [reply]
I successfully obtained Global Gamepus privileges:tada:.  Everything went smoothly.  Thank you very much for your assistance.:coffee:

**Reply by alu0101330105** [reply]
I checked now and seems to be solved for me too. Thank you very much.

**Reply by rust-kotlin** [reply]
Hi, @samus-aran , I have the same problem too. Could you give me the chance to re-apply? Thank you!

**Reply by ev1lQuark** [reply]
Hi, @samus-aran
I met the same problem, and I also need your help to re-verify my academic status. Thank you very much.😀

**Reply by JawadAsif** [reply]
Hi @samus-aran ,  I have the same problem. Can you please help. Hope you fix it soon. Thank you.

**Reply by jwzhi** [reply]
Hi @samus-aran, I have the same problem. I was approved 9 days ago but never able to get the benefits. Can you please help me in fixing this issue? Thank you.

**Reply by philipbl** [reply]
I am having the same problem but for faculty (https://github.com/orgs/community/discussions/67015). I got an early expiration notice but I can't re-apply because it says I already have the benefits (which I don't). I submitted a ticket 3 months ago and have not heard anything from GitHub on the issue.

**Reply by charlie-lindgren** [reply]
Edit: issue resolved for me!

I am also hoping to receive my benefits as well. I am approved (green) and waiting for benefits to be available. Any help to complete the process would be much appreciated. Thank you!

**Reply by LeonLicher** [reply]
I also have the same problem. I would be very grateful if you could help me get the benefits enabled. Thank You.

**Reply by Amad-Ahmed** [reply]
I am also having the same issue, was approved on 20th Nov but still have not received access.
![image](https://github.com/community/community/assets/80278397/57f0c31b-3a86-4340-8c45-3fd5a2dd8633)

**Reply by MarvAkhmed** [reply]
guys please who fixed this, what did you do to fix it just wait?

**Reply by GordonSlyding** [reply]
same with me :(

**Reply by MarvAkhmed** [reply]
okay now i can use students benifits, we just have to wait and it will be avilabel if the account is aproved

**Reply by NecessitasWeng** [reply]
![image](https://github.com/community/community/assets/146346767/3c022d8d-c6a2-48d3-ab79-d3c286b1dacf)
I have the same problem,was approved on 3rd Nov .@samus-aran - Can you please help me too?

**Reply by hemant-git123** [reply]
same problem with me

**Reply by HussAreus** [reply]
Same problem here. Shows approved, but cannot access benefits
![image](https://github.com/community/community/assets/71257590/77886749-0739-41c0-8e74-a274ee059af3)

**Reply by Felipe-Holanda** [reply]
![image](https://github.com/community/community/assets/93724958/adb3f60f-5240-49da-a425-5f6db166fa0e)

Same here too.

**Reply by UniNami** [reply]
![Screenshot from 2023-11-28 18-37-32](https://github.com/community/community/assets/104145863/10d48376-6e22-40e8-bb75-9591c1f1c41b)
Same here... no status update after 7 days - please help

**Reply by Lemonmos** [reply]
![82c3e1a040b7d05507c377fcf0fa520](https://github.com/community/community/assets/152026571/69700d91-b855-46b2-bc93-9ea5cbc5857d)
Hello @samus-aran , I have the same problem. I was approved in Nov 29, but I still have no access to benefits.  Could you help tackle this problem, thank you!

**Reply by Millitarychest** [reply]
Same problem... approved since september 30th
![image](https://github.com/community/community/assets/46942657/876457e8-eee3-4d28-b6b3-8c5683abd3da)

**Reply by rhettchase** [reply]
same issue 
![github](https://github.com/community/community/assets/126417892/2472d585-e991-481d-ad3f-59df51d217bf)

**Reply by fahim9778** [reply]
![Screenshot_20231206_012752_Chrome](https://github.com/community/community/assets/26436023/607668ab-d93e-4f97-9376-a61fc292f720)
I am having the same problem. If I am not mistaken, previously I would be verified and given access to Benefits like GitHub Copilot and others instantly. I don't know what changed and now I can't access these benefits.

@samus-aran , can you please help me regarding this?

**Comment by RPR21** [comment]
I am a verified student since October 1. I cannot redeem my student benefits. Can you help solve my problem?

**Reply by qijiabao4113** [reply]
@samus-aran  Can you please help this friend with the same problem too? ☕

**Reply by MarvAkhmed** [reply]
just wait and it will be available sense your account is approved

**Reply by junwenxiong** [reply]
It's been over 72 hours since my application, but I still can't redeem my student benefits. Can you help me?

**Reply by kozhinae** [reply]
Two weeks passed since my approval and I still don't get my benefits. What can I do to get it?

**Comment by Terror65workspace** [comment]
I have the same problem @samus-aran - Can you please help me too?

**Reply by MarvAkhmed** [reply]
just wait and it will be available sense your account is approved\

**Comment by HugoCodeX** [comment]
I am in the same situation, my plan to study is not available.

**Comment by nirajacharyaa** [comment]
### Dear GitHub Education Team, I hope this finds you well

- I have my student ID
- I also have my student's email
My campus is 7 8 km away from the main university premises which issued my ID and email, I did all the workaround because the camera won't switch to the back in my mobile device after I applied it said the campus doesn't appear in my country but it does, also I am not distant learning or anything.

Any help will be appreciated.
Regards.

**Reply by nirajacharyaa** [reply]
@samus-aran can you help ?
